thorlcr/thorutil/thormisc.cpp

```diff
@@ -99,6 +99,11 @@ const StatisticsMapping hashDistribActivityStatistics({StNumLocalRows, StNumRemo
 const StatisticsMapping nsplitterActivityStatistics({}, spillStatistics, basicActivityStatistics);
 const StatisticsMapping spillingWriteAheadStatistics({}, spillStatistics);
 
+const std::map<StatisticKind, StatisticKind> diskToSpillStatsMap
+={ {StSizeDiskWrite, StSizeSpillFile},
+   {StTimeDiskWriteIO, StTimeSpillElapsed}
```

Comment:
any reason not to add the rest at this juncture? e.g. the read ones
It will make the functionality more complete, even if the mapping where it is currently used won't currently use them yet, but will do when the mappings are update without any further changes.

Reply:
New stats will need to be created to map the remaining stats.  As far as I can see there isn't anything to map StCycleDiskReadIOCycles, StTimeDiskReadIO, StSizeDiskRead, StNumDiskReads and StNumDiskWrites.  Once temp equivalent stats for these are added, the statistic mapping will need to be updated too. If mappings aren't updated too, then there may be some unexpected error relating to merging stats that doesn't exist in the mapping.  It looks like too much work for this PR.

Reply:
ok, yes, agree